scripts/README.md

```diff
@@ -112,7 +131,7 @@ bash benchmark_serving_sharegpt.sh # to benchmark with ShareGPT dataset
 > The input/output ranges passed to `start_gaudi_vllm_server.sh` should cover the following benchmark ranges to get expected performance.
 
 > The parameters in the `benchmark_serving_range.sh` and `benchmark_serving_sharegpt.sh` must be modified to match the ones passed to `start_gaudi_vllm_server.sh`.
-### 3. Run vLLM with FP8 precision
+### 4. Run vLLM with FP8 precision
 Running vLLM with FP8 precision can be achieved using [Intel(R) Neural Compressor (INC)](https://docs.habana.ai/en/latest/PyTorch/Inference_on_PyTorch/Quantization/Inference_Using_FP8.html#inference-using-fp8) and by loading FP8 models directly (experimental).
```

Comment:
loading fp8 models directly is not supported in 1.21.0, please remove the corresponding readme.